docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -62,7 +62,7 @@ Here we use off-heap only.
 The starting point of every cache and also the faster since no serialization is necessary.
 You can optionally use copiers (see the section <<serializers-copiers.adoc#copiers,Serializers and Copiers>>) to pass keys and values by-value, the default being by-reference.
 
-A heap tier can be sized by entries or by size.
+A heap tier can be sized by entries or by size (though byte-based-sizing of heap tiers is now a deprecated feature).
```

Comment:
Let just drop this sentence. No need to note the deprecation if the sentence doesn't exist.

Reply:
just removed the comeplete sentence